index.html

```diff
@@ -11,9 +11,109 @@
   <![endif]-->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
 
 <body>
+
+
 <!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+<div class="container-services">
+  <!-- Copy and paste your HTML from the first UI project here -->
+    
+  
+  <body>
+  
+  <header class="head">
+      <nav class="navigation">
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+         <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#" class="borderbottom">Contact</a>
+      </nav> 
+
+    
+    <img href="index.html" class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    
+  
+    </header>
+  
+  <section class="main">
+  
+
+  <div class="lefts">
+    
+    <h1 class="innovation">Innovation</br>
+    On</br>
+    Demand</h1>
+    <button class="startedd">Get Started</button> 
+  
+    </div>
+    
+    <img class="codeimg" src="img/header-img.png" alt="Image of a code snippet." class="codepic">
+    
+  </section>
+  
+
+
+  <section class="firstmid">
+
+  <div class="seperator">
+    <h3>Features</h3>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  </p>
+  </div>
+
+  <div class="seperator">
+    <h3>About</h3>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  </p>
+</div>
+
+  </section>
+  
+
+  <section class="secondmid">
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+  </section>
+  
+    <section class="thirdmid">
+
+      <div class="threemid">
+    <h3>Services</h3>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  </p>
+  </div>
+
+  <div class="threemid">
+    <h3>Product</h3>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  </p>
+    </div>
+
+    <div class="threemid">
+  <h3>Vision</h3>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  </p>
+</div>
+  </section>
+  
+  <footer class="foot">
+  <h4>Contact</h4>
+  <p>
+    123 Way 456 Street</br>
+    Somewhere, USA</br>
+    1 (888) 888-8888</br>
+    <a href="mailto:sales@greatidea.io">sales@greatidea.io</a>
+  </p>
```

Comment:
```suggestion
  <p>
    123 Way 456 Street</br>
    Somewhere, USA</br>
    1 (888) 888-8888</br>
    <a href="mailto:sales@greatidea.io">sales@greatidea.io</a>
  </p>
```
I recommend wrapping this section in an `<address>` tag rather than a `<p>` tag because it's more semantic and will improve search engine optimization.